platform-sdk/swirlds-config-impl/src/main/java/com/swirlds/config/impl/internal/ConfigDataFactory.java

```diff
@@ -166,6 +170,15 @@ private <T> List<T> getDefaultValues(@NonNull final RecordComponent component) {
         if (Objects.equals(ConfigProperty.NULL_DEFAULT_VALUE, rawValue)) {
             return null;
         }
+
+        if (type.equals(CloudBucketConfig.class)) {
```

Comment:
This looks wrong. Can't we use converters?

Reply:
Connected to [this](https://github.com/hashgraph/hedera-services/pull/17010#discussion_r1891669593)